tests/e2e/paths/search/search.spec.js

```diff
@@ -442,8 +446,9 @@ test.describe('Path /search', () => {
         await expect(page.getByTestId('spatial-display_shapefile-name')).toHaveText('test.geojson')
         await expect(page.getByTestId('filter-stack-item__hint')).toHaveText('1 shape selected')
 
-        // Test leaflet has drawn the shape correctly
-        await expect(page.locator('.leaflet-interactive').first()).toHaveAttribute('d', 'M1161 404L1122 480L1257 499L1161 404z')
+        // Test leaflet has drawn the shape
+        await expect(await page.locator('g path').first()).toBeVisible()
+        await expect(await page.locator('g path').all()).toHaveLength(2)
```

Comment:
This polygon has 2 g paths?

Reply:
Interesting it does render both `geojson-svg leaflet-interactive` and a regular `leaflet-interactive` resolving comment

Reply:
The 2 paths come from the shapefile polygon being drawn, and the highlighted shape from the shape being selected